mender/templates/api-gateway-configmap.yaml

```
@@ -1,3 +1,5 @@
+{{ $entrypoint := ternary "https" "http" (.Values.api_gateway.env.SSL) }}
```

```suggestion
{{ $scheme := ternary "https" "http" (.Values.api_gateway.env.SSL) }}
```
with this single variable might refer to everywhere.

---

Comments accepted and addressed in commit 53d7e06.